bookingsRoutes.js

```diff
@@ -0,0 +1,131 @@
+const express = require("express");
+const bookings = require("./bookings.json");
+const fs = require("fs").promises;
+
+const bookingsRouter = express.Router();
+
+
+// ----------- Getting All -----------
+const getAllBookings = (req, res) => {
+  fs.readFile("./bookings.json", "utf8")
+    .then(bookings => {
+      bookings = JSON.parse(bookings);
+      res.status(200).json({ data: bookings });
+      return;
+    });
+};
+
+
+// ----------- Posting -----------
+const postBooking = (req, res) => {
+  const { title, firstName, surname, email, checkInDate, checkOutDate } = req.body;
+  if (!title.trim() || !firstName.trim() || !surname.trim() || !email.trim() || !checkInDate.trim() || !checkOutDate.trim()) {
+    res.status(400).json("Please Fill all the Fields!");
+    return;
+  }
```

Comment:
Nice validation here, although this could go into a separate function. For example imagine a:
```
const isEmailValid = (requestBody) => {
   const {title, firstName, surname, email, checkinDate, checkoutDate} = req.body
   return title.trim() && firstName.trim() && surname.trim() && email.trim() && checkInDate.trim() && checkOutDate.trim()) 
}
```

This moves the logic into a separate function, and converts the negatives into positives (which tend to be easier to understand when chaining boolean logic)